ballerina/natives.bal

```diff
@@ -23,8 +23,9 @@ import ballerina/jballerina.java;
 #
 # + input - An XML object, which needs to be transformed
 # + xsl - The XSL style sheet represented in an XML object
+# + params - Optional parameters passed into the xslt template
 # + return - The transformed result represented in an XML object or else an `error` if the given `XML` object
 #            cannot be transformed
-public isolated function transform(xml input, xml xsl) returns xml|error = @java:Method {
+public isolated function transform(xml input, xml xsl, map<string|int> params = {}) returns xml|error = @java:Method {
```

Comment:
Instead of the `int` type, isn't it better to have `decimal` type here?

Reply:
Updated the code to replace the `int` type with the `decimal` type.
[466c706](https://github.com/ballerina-platform/module-ballerina-xslt/pull/469/commits/466c706d9862d01056b7b081c6a5ab156a4ab739)